### DISCUSSION TITLE
Disable prompt for update the map

### DISCUSSION THREAD

**Discussion by thilokab** [osmandapp/OsmAnd#11089] [Q&A]
Hi Team,

Our users are prompting for a map update. 
It would be great if there is any way to disable that prompt for an update as it is consuming more data?

**Comment by vshcherb** [comment]
I think you can disable in Settings -> Dialogs and notifications ->  Download map dialog  / Startup message

**Comment by thilokab** [comment]
Thanks that worked really well.
We would like to know if this setting can be enforced through the OSM API. We are using OsmAndHelper.java within our application and would be great if this is supported through code.

Many Thanks

**Reply by vshcherb** [reply]
Related to https://github.com/osmandapp/OsmAnd/discussions/11088
